packages/terra-alert/tests/wdio/alert-spec.js

```diff
@@ -1,64 +1,61 @@
 Terra.describeViewports('Alert', ['tiny', 'large'], () => {
-  describe('Default', () => {
-    before(() => { browser.url('/#/raw/tests/terra-alert/alert/default-alert'); });
-    Terra.it.validatesElement();
+  it('displays default Alert', () => {
+    browser.url('/#/raw/tests/terra-alert/alert/default-alert');
+    Terra.validates.element('default');
   });
 
-  describe('Types', () => {
-    before(() => { browser.url('/#/raw/tests/terra-alert/alert/alert-type'); });
-    Terra.it.validatesElement();
-  });
+  it('displays types of Alert', () => {
+    browser.url('/#/raw/tests/terra-alert/alert/alert-type');
 
-  describe('Wrapped Content', () => {
-    before(() => { browser.url('/#/raw/tests/terra-alert/alert/alert-with-wrapped-content'); });
-    Terra.it.validatesElement();
+    Terra.validates.element('types');
   });
 
-  describe('Responsive', () => {
-    before(() => { browser.url('/#/raw/tests/terra-alert/alert/responsive-example'); });
-    Terra.it.validatesElement();
+  it('displays Alert with Wrapped Content', () => {
+    browser.url('/#/raw/tests/terra-alert/alert/alert-with-wrapped-content');
+
+    Terra.validates.element('wrapped content');
   });
 
-  describe('Titles', () => {
-    before(() => { browser.url('/#/raw/tests/terra-alert/alert/alert-title'); });
-    Terra.it.validatesElement();
+  it('displays Responsive Alert', () => {
+    browser.url('/#/raw/tests/terra-alert/alert/responsive-example');
+
+    Terra.validates.element('responsive');
   });
 
-  describe('Custom', () => {
-    before(() => { browser.url('/#/raw/tests/terra-alert/alert/custom-alert'); });
-    Terra.it.validatesElement();
+  it('displays Alert with titles', () => {
+    browser.url('/#/raw/tests/terra-alert/alert/alert-title');
+
+    Terra.validates.element('titles');
   });
 
-  describe('Action Button', () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-alert/alert/alert-action-button');
-    });
+  it('displays custom Alert', () => {
+    browser.url('/#/raw/tests/terra-alert/alert/custom-alert');
 
-    Terra.it.validatesElement();
+    Terra.validates.element('custom');
+  });
 
-    it('should be register actions', () => {
-      expect(browser.getText('#actionAlert')).to.have.string('clicked 0 times');
-      browser.click('#actionAlert button');
-      expect(browser.getText('#actionAlert')).to.have.string('clicked 1 times');
-    });
+  it('displays Alert with action button', () => {
+    browser.url('/#/raw/tests/terra-alert/alert/alert-action-button');
 
-    after(() => browser.moveToObject('#root', 0, 700));
+    Terra.validates.element('action button');
   });
 
-  describe('Dismissible', () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-alert/alert/alert-dismissible');
-    });
+  it('should register actions', () => {
+    expect(browser.getText('#actionAlert')).to.have.string('clicked 0 times');
```

Comment:
what page is this test run on? you should probably start it with a browser.url command or combine it with another it block.

Reply:
actually this is probably fine 🤷‍♂️